app/src/main/java/mozilla/lockbox/support/FxASyncDataStoreSupport.kt

```diff
@@ -32,26 +32,23 @@ class FxASyncDataStoreSupport(
         }
 
         // val encryptionKey = randomUUID().toString()
-        val encryptionKey = generateRandomString(50)
+        val encryptionKey = generateRandomString(32)
         preferences.putString(Constant.Key.encryptionKey, encryptionKey)
 
         encryptionKey
     }
 
-    /*
-     * This serves as a placeholder, and should definitely not be considered secure.
-     * TODO make this cryptographically interesting; add method to KeyStore/DataProtect
-     */
     private fun generateRandomString(keyLength: Int): String {
```

Comment:
IIUC, the returned string will contain this many *bytes of entropy*, but will not be this length.  Is this a potential API footgun where callers might expect the returned string to be of the specified length?

Reply:
your understanding is correct, I'll change the argument name